project.mml

```diff
@@ -677,6 +678,7 @@ Layer:
           FROM planet_osm_polygon
           WHERE highway IN ('pedestrian', 'footway', 'service', 'platform')
             OR (railway IN ('platform')
+            OR (tags->'area:highway') in ('footway', 'pedestrian')
```

Comment:
To powinno być pewnie tuż przed `ORDER BY COALESCE(layer,0), way_area DESC`, żeby się nie mieszało z tunelami i podziemnymi peronami..

Reply:
Wydaje mi się że mogło by być nawet linijkę wyżej, bo dotyczy w zasadzie identycznej funkcjonalności co linijka 679: `highway IN ('pedestrian', 'footway', 'service', 'platform')`. Tam też są wybierane powierzchnie dróg np. `highway=service`+`area=yes` i `highway=footway`+`area=yes`

Reply:
Masz rację, to chyba najlepsze miejsce.